Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -283,10 +283,11 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
 
                 method.setAttrs({
                     'multiple': this.get('multiple'),
-                    'loadContent': this.get('loadContent'),
+                    'loadContent': method.get('loadContent') === true ? true : this.get('loadContent'),
```

Comment:
that's a bit complicated to understand and kind of break the `loadContent` handling. Instead, you should force the `content` to be loaded internally but you should make sure the `loadContent` is correctly taken into account when giving back the selection.